src/components/Editor/Invitees/OrganizerList.vue

```diff
@@ -0,0 +1,81 @@
+<!--
+  - SPDX-FileCopyrightText: 2019 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<NcSelect class="organizer-select__multiselect"
+		:options="organizerSelection"
+		:searchable="false"
+		:max-height="600"
+		:placeholder="placeholder"
+		:class="{ 'showContent': inputGiven, 'icon-loading': isLoading }"
+		:clearable="false"
+		:label-outside="true"
+		:value="organizerSelected"
+		input-id="organizer"
+		label="label"
+		@option:selected="changeOrganizer">
+		<template #option="option">
+			<div class="organizer-select-list-item">
+				<!-- We need to specify a unique key here for the avatar to be reactive. -->
+				<Avatar :key="option.address"
+					:user="option.avatar"
+					:display-name="option.label" />
+				<div class="organizer-select-list-item__label">
+					<div>
+						{{ option.label }}
+					</div>
+					<div v-if="option.address !== option.label">
+						{{ option.address }}
+					</div>
+				</div>
+			</div>
+		</template>
+	</NcSelect>
+</template>
+
+<script>
+import {
+	NcAvatar as Avatar,
+	NcSelect,
+} from '@nextcloud/vue'
+
+export default {
+	name: 'OrganizerList',
```

Comment:
```suggestion
	name: 'OrganizerSelect',
```

Please also rename the file to `OrganizerSelect.vue`.